netty/shaded/src/testShadow/java/io/grpc/netty/shaded/ShadingTest.java

```diff
@@ -76,9 +79,8 @@ public void basic() throws Exception {
     server = ServerBuilder.forPort(0)
         .addService(new SimpleServiceImpl())
         .build().start();
-    channel = ManagedChannelBuilder
-        .forAddress("localhost", server.getPort())
-        .usePlaintext()
+    channel = Grpc.newChannelBuilder(
```

Comment:
I just realized: by replacing use of `ManagedChannelBuilder` or `NettyChannelBuilder` with `Grpc.newChannelBuilder` you have effectively removed the old tests for the old API. May be it's not a big deal because the API tests exist elsewhere and this file is just for testing shading. But just trying to point out.

Reply:
These particular tests are fine to only test one way. I've added a few more unit tests for the new codepath and added a basic test to interop-testing to test the old codepath.